core/src/components/range/range.ionic.scss

```
@@ -0,0 +1,216 @@
+  --height: 42px;
+
+  @include globals.typography(globals.$ion-body-md-regular);
+
+  z-index: globals.$ion-z-index-100;
+}
+
+:host(.range-item-start-adjustment) {
+  @include padding(null, null, null, globals.$ion-space-600);
+}
+
+:host(.range-item-end-adjustment) {
+  @include padding(null, globals.$ion-space-600, null, null);
+}
+
+:host(.ion-color) .range-bar-active,
+:host(.ion-color) .range-tick-active {
+  background: globals.current-color(base);
+}
+
+::slotted(ion-icon[slot]) {
+  font-size: globals.$ion-font-size-600;
+}
+
+::slotted([slot="start"]) {
+  @include margin(0, 16px, 0, 0);
+}
+
+::slotted([slot="end"]) {
+  @include margin(0, 0, 0, 16px);
+}
+
+::slotted([slot="label"]) {
+  max-width: globals.$ion-scale-5000;
+}
+
+:host(.range-has-pin:not(.range-label-placement-stacked)) {
+  /**
+  * The pin should not overlap any elements that are
+  * above the range. By adding padding to the top of the
+  * range, it provides a buffer for the pin to move into
+  * when it is pressed.
+  *
+  * The padding is not included when the label is stacked
+  * because the pin is below the label.
+  * It still requires a buffer to prevent the pin from
+  * overlapping the range. The buffer is added to the
+  * bottom of the range label instead of the host.
+  */
+  @include padding(calc(globals.$ion-space-200 + globals.dynamic-font(12px)), null, null, null);
+}
+
+:host(.range-has-pin.range-label-placement-stacked) .label-text-wrapper {
+  /**
+  * The pin should not overlap the stacked label. By adding
+  * margin to the bottom of the label, it provides a buffer
+  * for the pin to move into when it is pressed.
+  */
+  @include margin(null, null, calc(globals.$ion-space-200 + globals.dynamic-font(12px)), null);
+}
+
+.range-bar-active {
+  bottom: 0;
+
+  width: auto;
+
+  background: var(--bar-background-active);
+
+  &.has-ticks {
+    @include border-radius(0);
+    @include margin(null, calc(-1 * globals.$ion-scale-100 * 0.5), null, calc(-1 * globals.$ion-scale-100 * 0.5));
+  }
+}
+
+.range-tick {
+  @include margin-horizontal(calc(globals.$ion-scale-100 * -0.5), null);
+  @include border-radius(globals.$ion-border-radius-0);
+
+  position: absolute;
+  top: calc((42px * 0.5) - (globals.$ion-scale-300 * 0.5));
+
+  width: globals.$ion-scale-100;
+  height: globals.$ion-scale-300;
+
+  background: globals.$ion-primitives-neutral-100;
+  pointer-events: none;
+}
+
+.range-tick-active {
+  background: var(--bar-background-active);
+}
+
+.range-pin {
+  @include transform(translate3d(0, 100%, 0), scale(0.01));
+  @include padding(globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200, globals.$ion-space-200);
+
+  min-width: 28px;
+
+  transition: transform 120ms ease;
+
+  background: transparent;
+
+  font-size: globals.dynamic-font(12px);
+
+  text-align: center;
+}
+
+.range-knob {
+  border-width: globals.$ion-border-size-025;
+  border-style: solid;
+  border-color: globals.ion-color(primary, base);
+}
```

```suggestion
    box-sizing: border-box;

}
```

This will ensure it remains `24px` width/height even with the border

You might have to run `npm run lint` after I am not sure if this is the right order